### PR TITLE
#FN:16 chore: Setting up an empty react project

### PR DESCRIPTION
**Pull request by kunda4**
# PR Description
The team should create a react project from scratch using Vite, and linting with TSlint.
Please include a brief description of the pull request you want to raise.
# Description of tasks that were expected to be completed
List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style
- [x] Create an empty react project using vite
- [x] linting it with TSlint.
# Description of tasks that were expected to be completed and that are not yet completed
- [x] None
# How this may be tested?
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
1. run `yarn i` to install all required dependencies
2. run `yarn run dev`
3. run `yarn run lint`
4. step 4...
# Screenshots (If appropriate/applicable)
Provide screenshots related to your PR in case it's appropriate and applicable
# Please check this Checklist before you submit your PR:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by dushimeemma — COMMENTED** [review]
@kunda4 would you mind to use `yarn` package manager?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```
Comment (dushimeemma) [inline index.html]: change the title of the project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```
Comment (dushimeemma) [inline index.html]: change favicon of the project.

File: src/assets/react.svg
Comment (dushimeemma) [inline src/assets/react.svg:1]: remove this image.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "ecommerce-app-cogito-fe",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```
Comment (dushimeemma) [inline package.json:7]: add start script as well.

**Review by dushimeemma — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kunda4) [reply index.html]: Done

**Review by Niyobuzima — APPROVED** [review]
LGTM!!!!

**Review by Honore123 — APPROVED** [review]
🚀

File: .github/Pull-Request-Template [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] Create .....
+- [ ] Add .....
+- [ ] ......
+# Description of tasks that were expected to be completed and that are not yet completed
+- [ ] Create .....
+- [ ] Add .....
+- [ ] ......
+# How this may be tested?
+Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
+1. run `npm i` to install all required dependencies
```
Comment (dushimeemma) [inline .github/Pull-Request-Template]: update this file and use yarn as well @kunda4

**Review by dushimeemma — CHANGES_REQUESTED** [review]

**Review by uwdjany — APPROVED** [review]
Well done!

File: .github/Pull-Request-Template (hunk 4 above)
Comment (kunda4) [reply .github/Pull-Request-Template]: done

File: .github/Pull-Request-Template
Comment (dushimeemma) [inline .github/Pull-Request-Template:1]: Would you mind to use a markdown file for PR template?

**Review by dushimeemma — CHANGES_REQUESTED** [review]

**Review by dushimeemma — APPROVED** [review]
Good job 🥇

**Pull request merged**